test/parallel/test-http2-connect-options.js

```diff
@@ -12,7 +12,8 @@ const assert = require('assert');
 
 const server = http2.createServer((req, res) => {
   console.log(`Connect from: ${req.connection.remoteAddress}`);
-  assert.strictEqual(req.connection.remoteAddress, '127.0.0.2');
+  assert.match(req.connection.remoteAddress,
+               /^127\.0\.0\.1$|^127\.0\.0\.2$|^::1$/);
```

Comment:
Should the `127.0.0.1` address be accepted here? It wasn't before.
```suggestion
  assert.match(req.connection.remoteAddress, /^(127\.0\.0\.2|::1)$/);
```